tuna/connection.py

```diff
@@ -153,35 +176,45 @@ def exec_command_unparsed(self, cmd, timeout=SSH_TIMEOUT, abort=None):
       stdout, stderr = self.subp.stdout, self.subp.stderr
       return 0, stdout, stderr
 
+    cmd_idx: int
     for cmd_idx in range(NUM_CMD_RETRIES):
       try:
         if self.ssh is None or not self.ssh.get_transport(
         ) or not self.ssh.get_transport().is_active():
           self.ssh_connect()
+        i_var: Any
+        o_var: Any
+        e_var: None
         i_var, o_var, e_var = self.ssh.exec_command(cmd, timeout=timeout)
+
       except Exception as exc:
         self.logger.warning('Machine %s failed to execute command: %s', self.id,
                             cmd)
         self.logger.warning('Exception occurred %s', exc)
         self.logger.warning('Retrying ... %s', cmd_idx)
-        retry_interval = randrange(SSH_TIMEOUT)
+        retry_interval = randrange((int)(SSH_TIMEOUT))
```

Comment:
Was there an issue where SSH_TIMEOUT needed to be cast to an int everywhere?
Perhaps change 60.0 to 60 at the top of the file?

Reply:
correted